unison-src/transcripts/docs.output.md

```diff
@@ -153,14 +153,14 @@ Now that documentation is linked to the definition. We can view it if we like:
   
   ## Examples:
   
-  List.take.ex1 = builtin.List.take 0 [1, 2, 3, 4, 5]
+  ex1 = builtin.List.take 0 [1, 2, 3, 4, 5]
   🔽
-  List.take.ex1 = []
+  ex1 = []
   
   
-  List.take.ex2 = builtin.List.take 2 [1, 2, 3, 4, 5]
+  ex2 = builtin.List.take 2 [1, 2, 3, 4, 5]
   🔽
-  List.take.ex2 = [1, 2]
+  ex2 = [1, 2]
```

Comment:
This looks like the LHS is using the short name when it should use the full name?

Reply:
This is in the documentation printer.

Reply:
Yeah, this is a bug... taking a look.

Reply:
Okay fixed.